hosted/aks/helper/helper_cluster.go

```diff
@@ -472,6 +472,23 @@ func CreateAKSClusterOnAzure(location string, clusterName string, k8sVersion str
 	return nil
 }
 
+// ClusterExistsOnAzure gets a list of cluster based on the name filter and returns true if the cluster is not in Deleting state;
+// it returns false if the cluster does not exist or is in Deleting state.
+func ClusterExistsOnAzure(clusterName, resourceGroup string) (bool, error) {
+	fmt.Println("Showing AKS cluster ...")
+	args := []string{"aks", "show", "--subscription", subscriptionID, "--name", clusterName, "--resource-group", resourceGroup}
+	fmt.Printf("Running command: az %v\n", args)
```

Comment:
Can the command output be _quiet_ ? to prevent sensitive information display in logs.
Rest changes lgtm.